docs/docusaurus.config.js

```diff
@@ -3,15 +3,15 @@ module.exports = {
   url: "https://lyradocs.netlify.com",
   baseUrl: "/",
   onBrokenLinks: "throw",
-  favicon: "img/favicon.ico",
+  favicon: "./images/logo.png",
```

Comment:
these placeholder images needed to be moved into a folder called `static`, which apparently is a ⭐magic ⭐ folder docusaurus needs all static assets to be in